app/views/homes/show.html.erb

```diff
@@ -4,9 +4,18 @@
     Hotline Webring
     <a href="https://hotlinewebring.club/hotlinewebring/next">&rarr;</a>
   </h2>
+  <% if Rails.configuration.closed %>
+    <h2>Is Closed for Registration</h2>
```

Comment:
"Closed For Registration" is slightly awkward phrasing to me. What about "Is Not Accepting New Sites"?

Screenshot:
![CleanShot 2023-12-15 at 11 03 45@2x](https://github.com/hotline-webring/hotline-webring/assets/257678/eba5e13c-30ba-45b4-882f-d94378480e15)

Reply:
Also: we use the `<h2>` and `<h3>` text here and in the `_closed` partial. Can you move it to I18n so we can do something like:

```erb
<%= t("closed_for_registration.subheader") %>
```

and

```erb
<%= t("closed_for_registration.existing_sites_still_work") %>
```

and use that in both places?